src/modules/index.js

```diff
@@ -1 +1,90 @@
-import '../styles/style.scss';
\ No newline at end of file
+import '../styles/style.scss';
+// Initialize DOM elements of interest
+const tableBody = document.getElementById('table-body');
+const formName = document.getElementById('name');
+const formScore = document.getElementById('score');
+const sumbitButton = document.getElementById('submit');
+const form = document.getElementById('form');
+const refreshButton = document.getElementById('refresh-btn');
+// Initialize app data
+let GAME_ID = null;
+const BASE_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+// This function creates a single child table data node
+const singleView = (name, score) => {
+  const row = document.createElement('tr');
+  const data = document.createElement('td');
+  data.innerHTML = `${name}: ${score}`;
+  row.append(data);
+  tableBody.append(row);
+};
+
+// This sets up the game and keeps track of it
+const createGame = async () => {
+  const localData = localStorage.getItem('leaderboardGameId');
+  if (localData) {
+    GAME_ID = localData;
+  }
+  if (!localData) {
+    GAME_ID = await fetch(`${BASE_URL}games`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        name: 'Leaderboard Game',
+      }),
+    }).then((res) => res.json()).then((data) => data.result.split(' ')[3]);
+    localStorage.setItem('leaderboardGameId', GAME_ID);
+  }
+};
+
+// This populates the DOM with updated content
+const updateViews = (data) => {
+  tableBody.innerHTML = '';
+  for (let i = 0; i < data?.length; i += 1) {
+    const { user, score } = data[i];
+    singleView(user, score);
+  }
+};
+
+// This fetches updated data from the api
+const refresh = async () => {
+  await fetch(`${BASE_URL}games/${GAME_ID}/scores`, {
+    method: 'GET',
+    headers: { 'Content-Type': 'application/json' },
+  }).then((res) => res.json()).then((data) => updateViews(data.result));
+};
+
+// This handles creation of new records
+sumbitButton.addEventListener('click', async (e) => {
+  e.preventDefault();
+  const user = formName.value;
+  const score = formScore.value;
+  await fetch(`${BASE_URL}games/${GAME_ID}/scores`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user,
+      score,
+    }),
+  }).then(() => {
+    form.reset();
+    refresh();
+  });
+});
+
+// Triggers a refresh of data;
+refreshButton.addEventListener('click', () => refresh());
+
+// Initializes the application
+const initialize = () => {
+  form.reset();
+  createGame();
+  refresh();
+};
+
+// Starts instance
+initialize();
```

Comment:
- Good job done so far, but your app has some unexpected behavior, I get a different list of scores when the app renders initially and another list of scores when I submit a new score. A likely cause for this is that you are generating a new GAME_ID when the app renders, I suggest using the same GAME_ID in your project. Please fix this 🙏 

Your project(when it renders initially):
<img width="563" alt="a35ya35ya35y" src="https://user-images.githubusercontent.com/73607512/224406735-cd38be5e-d7b2-42f3-b91a-02a97502f255.PNG">

Your project(after submitting a new score):
<img width="576" alt="srtjsrtjsrtj" src="https://user-images.githubusercontent.com/73607512/224406821-84fed4e8-17eb-4a07-bed3-070104909e76.PNG">